utils/cdk-runner/pkg/aws/cloudformation/stack.go

```diff
@@ -103,6 +104,11 @@ func StackOperationInProgress(c *Client, stackName string) (bool, string, error)
 }
 
 func (s *CfnStack) LogEvents(c *Client) {
+	if os.Getenv(DryRunEnvKey) == "true" {
```

Comment:
Can this be put as a property of the client to avoid getting the env var everywhere?

Reply:
It could be added as a property of the client but it's only called once after the client is created. I've added function instead.